tests/panic_source_locations.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(unused_imports)]
```

Comment:
These `unused_imports` attributes (and the `dead_code` attribute below) are to silence warnings for rustc-versions `< 1.46`. Is there a better way to do this?

Reply:
Can you make the imports conditional?

```rust
#[rustversion::since(1.46)]
use foo;
```